modules/core/react/spec/InputProvider.spec.tsx

```diff
@@ -208,7 +208,8 @@ describe('InputProvider', () => {
         </InputProvider>
       </InputProvider>
     );
-    expect(component.find('.wdc-input-provider').length).toBe(1);
```

Comment:
Was this class name used for anything other than nested detection?

Reply:
i don't think so!